internal/accounts/account-add.go

```diff
@@ -0,0 +1,88 @@
+package accounts
+
+import (
+	"bytes"
+	"fmt"
+	"text/tabwriter"
+
+	"github.com/onflow/flow-cli/internal/command"
+	"github.com/onflow/flow-cli/pkg/flow"
+	"github.com/onflow/flow-cli/pkg/flow/services"
+	"github.com/spf13/cobra"
+)
+
+type flagsAdd struct {
+	KeySigAlgo  string `default:"ECDSA_P256" flag:"sig-algo" info:"Signature algorithm"`
+	KeyHashAlgo string `default:"SHA3_256" flag:"hash-algo" info:"Hashing algorithm"`
+	KeyIndex    int    `flag:"index" info:"Account key index"`
+	KeyHex      string `flag:"privateKey" info:"Private key in hex format"`
```

Comment:
```suggestion
	KeyHex      string `flag:"private-key" info:"Private key in hex format"`
```

Other flags are hyphenated instead of camelcase. Is that true across the whole CLI?

Reply:
good catch, I will make sure all are hyphenated